qemu/tests/cfg/x86_cpu_flags.cfg

```
@@ -124,3 +121,17 @@
                     test_dir = '/home'
                     source_file = 'cpuid.src.tar.gz'
                     check_cpuid = './cpuid -r | grep eax=0x40000001 | uniq -c'
+                - memory_protection_key:
+                    flags = "pku ospke"
+                    variants:
+                        - @default_model:
+                            cpu_model_flags += ",+pku"
+                            HostCpuVendor.amd:
+                                no RHEL.6
+                                no Host_RHEL.m8.u2
+                        - host_model:
+                            only HostCpuVendor.amd
+                            no RHEL.6
+                            no Host_RHEL.m8.u2
```

```suggestion
                    flags = "pku ospke"
                    HostCpuVendor.amd:
                        no RHEL.6
                        no Host_RHEL.m8.u2
                    variants:
                        - @default_model:
                            cpu_model_flags += ",+pku"
                        - host_model:
                            only HostCpuVendor.amd
```
And, what about RHEL.7 and Host_RHEL.m8.u1?

---

@vivianQizhu update, thanks.